src/components/SwipeableView/index.js

```diff
@@ -1,3 +1,36 @@
-export default ({children}) => children;
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Pressable} from 'react-native';
 
-// Swipeable View is available just on Android/iOS for now.
+const propTypes = {
+    /** this could be set to provide events */
+    onPress: PropTypes.func,
+
+    /** can be set to provide styles */
+    styles: PropTypes.arrayOf(PropTypes.shape({})),
+
+    /** Children to render. */
+    children: PropTypes.oneOfType([
+        PropTypes.func,
+        PropTypes.node,
+    ]).isRequired,
+};
+
+const defaultProps = {
+    onPress: () => {},
+    styles: [],
+};
+
+// Swipeable View is available just on Android/iOS for now. Still need to utilize events
+function SwipeableView(props) {
+    return (
+        <Pressable style={props.styles} onPress={props.onPress}>
+            {props.children}
+        </Pressable>
+    );
+}
+
```

Comment:
`onPress` & `Pressable` in SwipeableView doesn't make any sense

Reply:
I need this here to handle the press event, it doesn't work outside this component

Reply:
Why do we need to modify this `SwipeableView`? 

If we need to listen to multiple gestures then it should be defined separately. This is something we would require only in `AttachmentCarousel`, so it's better to write there.

cc: @chiragsalian

Reply:
@marcaaron What's your take on this

Reply:
Ah sorry, I'm not able to answer every question as I'm not reviewing this PR or up to speed on what we are trying to do here. If you have a more general question about something feel free to post a summary in Slack.

Reply:
Thanks @marcaaron, will do that!